src/modules/wallet/utils/multiSignatureStatus.js

```diff
@@ -7,14 +9,26 @@ import {
 /**
  * Get required statuses for a given multi-signature transaction
  */
+// eslint-disable-next-line max-statements
 export const getMultiSignatureStatus = ({
   senderAccount,
   account,
-  transactionJSON,
   currentAccount,
+  moduleCommandSchemas,
+  transactions,
+  ...rest
```

Comment:
Why not just add `transactionJSON` here instead of ...rest so it is easy to see exactly what this function requires

Reply:
reason was because I wanted to preserve the term transactionJSON through in the code snippet, defining it there would mean that I had to define a new variable name which I didn't want to do.

Reply:
Dont you think it is easier to understand the function just renaming in those cases?
`transactionJSON: initialTransactionJSON`

This way you see exactly what the function needs (gets help in editors etc...) as well as you know that this var is both modified and used in its initialState.